### PR TITLE
targets: merge js2c.py into targets/tools folder

### PR DESCRIPTION
**Pull request by seanshpark**
* also add .PHONY to target makefiles

JerryScript-DCO-1.0-Signed-off-by: SaeHie Park saehie.park@samsung.com

### CODE REVIEW

File: targets/tools/js2c.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def removeWhitespaces(code):
     return re.sub('\n+', '\n', re.sub('\n +', '\n', code))
 
 
-LICENSE = '''/* Copyright 2015 Samsung Electronics Co., Ltd.
+LICENSE = '''/* Copyright 2016 Samsung Electronics Co., Ltd.
```
Comment (egavrin) [inline targets/tools/js2c.py]: I think it should be `2015-2016` too.

**Comment by egavrin** [comment]
LGTM

**Comment by galpeter** [comment]
lgtm

**Comment by seanshpark** [comment]
thanks for quick review :)

**Pull request merged**